solr/core/src/test/org/apache/solr/cli/PackageToolTest.java

```diff
@@ -263,7 +264,7 @@ public void testPackageTool() throws Exception {
               "question-answer",
               "--collections",
               "abc",
-              "-p",
+              "--params",
```

Comment:
Should not this be `--param`?

Reply:
It should, but we're going to do a separate pass on all the variations of this property!  Checking the spreadsheet we have:  
```

-params
--
--params
--param
--parameters
```

Reply:
Package tool has: ```
Option.builder()
            .longOpt("param")
            .hasArgs()
            .argName("PARAMS")
            .desc("List of parameters to be used with deploy command.")
            .build(),
```